beacon-chain/rpc/prysm/v1alpha1/validator/proposer_attestations.go

```diff
@@ -409,15 +411,15 @@ func (a proposerAtts) dedup() (proposerAtts, error) {
 }
 
 // This filters the input attestations to return a list of valid attestations to be packaged inside a beacon block.
-func (vs *Server) validateAndDeleteAttsInPool(ctx context.Context, st state.BeaconState, atts []ethpb.Att) ([]ethpb.Att, error) {
+func (vs *Server) validateAndDeleteAttsInPool(ctx context.Context, st state.BeaconState, atts []ethpb.Att) []ethpb.Att {
```

Comment:
I changed the signature of this function because not being able to delete an attestation from the pool should not stop proposing